global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "7.0.100",
+    "dotnet": "8.0.100-alpha.1.23055.1",
```

Comment:
I am wondering if we should already be building with such a runtime. Should we adhere to release/7.0 arcade until the toolset stabilizes a little?

Reply:
I do need the 8.0 SDK to switch to testing on 8.0 and I would assume that if arcade starts using it, it should be stable, but it seems the new SDK broke all the single-file testing even against 6.0/7.0 here.

Reply:
Actually it may not be the 8.0 SDK as much as the newer arcade version because in my test against 8.0 branch, the single-file tests pass (on all version on Windows, but fail on 8.0 Linux).  I still get a bunch of invalid heap obj/verifyheap failing in the non-single-file 8.0 runtime that I need to figure out.

Reply:
They don't assume it's stable - they just assume repos are not releasing bits.